src/Sylius/Component/Core/Checker/Eligibility/PromotionCouponChannelEligibilityChecker.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Checker\Eligibility;
+
+use Sylius\Component\Core\Model\OrderInterface as ModelOrderInterface;
+use Sylius\Component\Core\Model\Promotion as CoreModelPromotion;
```

Comment:
```suggestion
use Sylius\Component\Core\Model\PromotionInterface;
```
Also, update the spec accordingly since ATM [this](https://github.com/Sylius/Sylius/pull/14237/files#diff-b74404a6c934929cd059bebd60336c57b94e30ac0655133f12c60f6073293a34R74) is just confusing